### ISSUE TITLE
could you make a flatpak for playmyvideos?

### ISSUE THREAD

**Issue by altntowin** [artemanufrij/playmyvideos#15]
Hi, I really appreciate your work , is a good application
but i don't use elementary OS
could you make a flatpak for playmyvideos?
in this way, users can install the application in any linux distribution

please.
thank you.

**Comment by artemanufrij** [comment]
https://github.com/artemanufrij/playmyvideos#install-from-github

**Comment by worldofpeace** [comment]
Hi @altntowin 
There's been work done for this in [playmymusic](https://github.com/artemanufrij/playmymusic/issues/55) by @bil-elmoussaoui.
He also said:

> Once this one is done, I will package the playmyvideos too :)
>
So expect this hopefully soon...

**Comment by bil-elmoussaoui** [comment]
Yeah I can work on that 👍  I'm still waiting for some fixes on playmymusic so I can update the flatpak package

**Comment by artemanufrij** [comment]
@bil-elmoussaoui look here: https://github.com/flathub/flathub/pull/229#issuecomment-360982290

I need some information for fix appdata errors. Could you post me a link?

**Comment by bil-elmoussaoui** [comment]
The Invalid style are not errors :) the missing tags are

**Comment by artemanufrij** [comment]
@bil-elmoussaoui thanks for feedback. I will fix it :+1:

**Comment by artemanufrij** [comment]
@bil-elmoussaoui no more missing tags. fixed in here and in https://github.com/artemanufrij/playmymusic/issues/55

**Comment by artemanufrij** [comment]
@bil-elmoussaoui I will switch to meson after Juno release... have you a simple example for me how I can to create a cmake file?

**Comment by bil-elmoussaoui** [comment]
Check out FeedReader source code. All you need is to copy the files I added as a patch to my PR in flathub and add that line I mentioned on the other issue which let cmake check that folder first for the required cmake modules

**Comment by artemanufrij** [comment]
@bilelmoussaoui I switched to meson. What are the next steps for creating a flatpak?

**Comment by bilelmoussaoui** [comment]
@artemanufrij can you ping me somewhere so I can build a Flatpak package for this? I might need some help with testing/dependencies :)

**Comment by artemanufrij** [comment]
@bilelmoussaoui ping :).
feel free to build a package :+1: if you need, so let me know...
